### PR TITLE
[CDAP-17444] Replication add metrics table list view

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://cdap.atlassian.net/browse/CDAP-17444
Build: https://builds.cask.co/browse/CDAP-UDUT892

### CODE REVIEW

File: cdap-ui/app/cdap/components/Replicator/Detail/TableScatterPlotGraph/ScatterPlotTable.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            );
+          })}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+function numberFormat(d) {
+  return numeral(d).format('0.00a');
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Replicator/Detail/TableScatterPlotGraph/ScatterPlotTable.tsx]: Should we just move this to the helper function (`humanReadableNumber`). Right now we have a specific function for bytes, may be we could introduce one for decimal with units?

**Review by ajainarayanan — APPROVED** [review]
One minor comment. 👍 LGTM

File: cdap-ui/app/cdap/components/Replicator/Detail/TableScatterPlotGraph/ScatterPlotTable.tsx (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Replicator/Detail/TableScatterPlotGraph/ScatterPlotTable.tsx]: moved to a helper function

**Pull request merged**